underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystemFactory.java

```diff
@@ -47,7 +47,7 @@ public UnderFileSystem create(String path, Object unusedConf) {
 
     if (addAndCheckGoogleCredentials()) {
       try {
-        return new GCSUnderFileSystem(new AlluxioURI(path));
+        return GCSUnderFileSystem.createGCSUnderFileSystem(new AlluxioURI(path));
```

Comment:
the naming here is a bit redundant, maybe switch to just `GCSUnderFileSystem.create(new AlluxioURI(Path))`?